tests/cancun/eip1153_tstore/test_tstorage_execution_contexts.py

```diff
@@ -308,7 +308,7 @@ def tx(pre: Alloc, caller_address: Address, callee_address: Address) -> Transact
     return Transaction(
         sender=pre.fund_eoa(),
         to=caller_address,
-        data=left_pad_zeros_up_to_size(callee_address, 32),
+        data=Hash(callee_address, left_padding=True),
```

Comment:
again the number 32 is implicit

Reply:
Yes, but I think at this point the tester has already selected `Hash` so they know they needed a 32-byte constructor.

Another entirely different discussion could be that the hashes become longer in the future, but in that case I think we need a completely new class.